iothub/service/src/Jobs/JobProperties.cs

```diff
@@ -143,6 +151,12 @@ public JobProperties()
         [JsonProperty(PropertyName = "configurationsBlobName", NullValueHandling = NullValueHandling.Ignore)]
         public string ConfigurationsBlobName { get; set; }
 
+        /// <summary>
+        /// Convenience property to determine if the job is in a terminal state, based on <see cref="JobStatus"/>.
+        /// </summary>
+        [JsonIgnore]
+        public bool IsFinished => s_finishedStates.Contains(Status);
```

Comment:
IsComplete? IsTerminal?

Reply:
Given that "Complete" is its own status, I like "IsFinished".